lib/game.js

```diff
@@ -0,0 +1,59 @@
+const Board = require('./board.js');
+const Player = require('./player.js');
+
+module.exports = class Game {
+  constructor(playerX, playerO) {
```

Comment:
Does this class still work if I flip the player when creating a new instance
`const game = new Game(playerX, playerO);` vs `const game = new Game(playerO, playerX);`

If it doesn't, I would use a more abstract name